lib/matplotlib/tight_layout.py

```diff
@@ -1,3 +1,16 @@
 from matplotlib._tight_layout import *  # noqa: F401, F403
 from matplotlib import _api
 _api.warn_deprecated("3.6", name=__name__, obj_type="module")
+
+
+@_api.deprecated("3.6", alternative="figure.canvas.get_renderer()")
```

Comment:
I don't think this hurts anything, and may be the only reason some folks were importing the module.